### ISSUE TITLE
SOCKS access for Eric Murphy

### ISSUE THREAD

**Issue by murphytwin** [department-of-veterans-affairs/va.gov-team#40085]
### Your Name

Eric Murphy

### Your Email

eric.murphy@adhocteam.us

### Your Role

UX Designer

### Public SSH Key

ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDKY4dXFnlDjE0lP0s+LlrUNgFZRjxa4q7VgO8m3rfla13AT1RUrzEquvv+7WobCqTdOYGaEbDfrvEauGvVxoP3s1HwvPApkpoJ1I6tZ1o46rVbR/Mkudj04Qp1m/R9T1VYp4T8nAoDt0ZPaKoJ0G/zQ+Ze7L0ul0ioCQl0FPbugY3jpZQc7MD8GKLp/Gi6Sv+U7QoUYb2ePlcz+31dGSaK+ogURcptE8Jnqm5k5x7ul6RB0hnht7h7O8oq/bLllsFD+O6IO8PPUfxJJ1Bgz9ZzbUXXp1mx2JVIRtZMjJT6ECsmSq7Mj0Z0EZUkTZfZgCCFteFaw5u16pYpxQ2FhA9u5RFteli5kAXURqXQ196wGZUZ6lctEHsXCykyQuQ3z3Jsw17sUe1RqXxndjv39zrUFz2xlW6wBGEz+YNC4DmEo0iRXsHS2ulY9L5McVEHEs8g0zosbHNgTsJ1zf6KfE/Da0Vv8vLvsz2MUMo4yNqytctzLv0EhPVEZl5IxYEHLIk= ericmurphy@ericmurphysmbp.la

**Comment by rmtolmach** [comment]
@murphytwin Access granted ✅ 

Please allow 10 minutes for your key to be propagated to the various locations before attempting to connect.
See docs for further steps: https://depo-platform-documentation.scrollhelp.site/getting-started/Internal-tools-access-via-SOCKS-proxy.1821081710.html

Reach out in #vfs-platform-support if you have any trouble.
